deploy/operator.yaml

```diff
@@ -61,16 +61,23 @@ spec:
               value: "False"
             - name: PROFILER_PORT
               value: "6060"
+            - name: NMSTATE_INSTANCE_NODE_LOCK_FILE
+              value: "/etc/k8s_nmstate/handler_lock"
           volumeMounts:
           - name: dbus-socket
             mountPath: /run/dbus/system_bus_socket
+          - name: nmstate-lock
+            mountPath: /etc/k8s_nmstate
           securityContext:
             privileged: true
       volumes:
       - name: dbus-socket
         hostPath:
           path: /run/dbus/system_bus_socket
           type: Socket
+      - name: nmstate-lock
+        hostPath:
+          path: /etc/k8s_nmstate
```

Comment:
Who is creating this dir ?

Reply:
It should be created automatically (by the lock library), BTW, I run the following tests on local-cluster [1]:
A. run nmstste-handler deployment  - looks OK  (directory+file created)
B. ssh to node and delete nmstste-handler container (using docker ), a new container was created - looks OK

I guess we should add a test that runs two nmstate-handlers concurrently - maybe we can have it in a follow-up PR. 

[1] https://github.com/nmstate/kubernetes-nmstate/blob/master/docs/deployment-local-cluster.md